### DISCUSSION TITLE
The webhook configuration for version 9.7.0 is not detailed

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11656] [General]
skywalking version; 9.7.0
The webhook configuration is not detailed in the alarm manual on the official website.
Can you give an example of webhook configuration?

Is this configuration correct?
` ` `
webhooks:
is-default: true
webhooks:
- http://10.10.255.226:8680/send
` ` `
Or is this configuration correct? And the oap log does not print the sent log
` ` `
webhook:
- http://10.10.255.226:8680/send
` ` `

**Comment by wu-sheng** [comment]
```yaml
#  webhook:
#    default:
#      # If true, this hook will apply on all rules, unless a rule has its own specific hook. Could have more than one default hooks in the same hook type.
#      is-default: true
#      urls:
#        - http://127.0.0.1/notify/
#        - http://127.0.0.1/go-wechat/
#
#  gRPC:
#    default:
#      is-default: true
#      target-host: 127.0.0.1
#      target-port: 9888
```

These should be the right way. @wankai123 Could you update the docs? I noticed this exists in `alarm-settings.yml` only.

**Reply by wankai123** [reply]
updated, https://github.com/apache/skywalking/pull/11658

**Reply by 1450793561** [reply]
OK

**Comment by 1450793561** [comment]
done
